apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/CarouselConcert.test.tsx

```
@@ -0,0 +1,393 @@
+import React from 'react';
+import { render, screen, fireEvent, act } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { CarouselConcert } from '@/components/CarouselConcert';
+
+// Mock images array for consistent testing
+const mockImages = [
+    "https://as2.ftcdn.net/v2/jpg/10/62/03/07/1000_F_1062030789_yaKuxV8QuIRqIgAfTA0wYlc797DinmdH.jpg",
```

ESLint

[error] Found a string with entropy 4.54 : "https://as2.ftcdn.net/v2/jpg/10/62/03/07/1000_F_1062030789_yaKuxV8QuIRqIgAfTA0wYlc797DinmdH.jpg"

(no-secrets/no-secrets)